### PR TITLE
Fix spacing issues on landing page cards

### PR DESCRIPTION
**Pull request by cesarvarela**

### CODE REVIEW

File: site/gatsby-site/src/pages/landingPage.js [hunk 1]
```diff
@@ -37,7 +37,17 @@ const StyledCardParagraph = styled.p`
 `;
 
 const StyledRow = styled(Row)`
-  margin-bottom: 1em;
+  & + & {
```
Comment (smcgregor) [inline site/gatsby-site/src/pages/landingPage.js:40]: What is this `& + &` syntax doing?

**Comment by smcgregor** [comment]
I have one question regarding syntax, but this looks visually much improved and the implementation looks good. Merging now.

File: site/gatsby-site/src/pages/landingPage.js (hunk 1 above)
Comment (cesarvarela) [reply site/gatsby-site/src/pages/landingPage.js:40]: The `&` gives you access to the class name that styled-components will generate for the current component (kind of the `this` keyword). The `+` is just the adjacent sibling selector

Comment (cesarvarela) [reply site/gatsby-site/src/pages/landingPage.js:40]: It does look cryptic tho

**Pull request merged**